tests/test_lightning_integration.py

```diff
@@ -1,4 +1,4 @@
-from lightning.cli.lightning_cli import run_app
+from lightning_app.cli.lightning_cli import run_app
```

Comment:
@tchaton should it be `lightning.app` or `lightning_app` here?

Reply:
the lightning repo has a lot of references to `lightning_app`

Reply:
It's better to be lighting.app